tags_handling.py

```diff
@@ -58,3 +61,11 @@ def get_file_tags(file_path):
             "album_title": album_title_tag(file_path, audio),
             "year": year_tag(file_path, audio)}
     return tags
+
```

Comment:
To jeszcze będzie jeden kwadracik do tego zadania - usuń `get_file_tags` i wszystkie funkcje które nie będą już używane. Czyli krótko mówiąc sprzątanie.

Reply:
Nie zapomnij po sprzątaniu sprawdzić czy testy wciąż przechodzą.